src/components/Settings/Speech/Speech.container.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
```

Comment:
File should be renamed to `Speech.component`

Reply:
I've fixed the names in https://github.com/shayc/cboard/pull/134/commits/1627dac4bbe6678a5f5a5c879fd3ed6991d274e6. Sorry for the confusion. :sweat_smile: